guide/C/ch_reports.xml

```diff
@@ -88,6 +88,28 @@ Configuration.</para>
 price information in your file, the report will indicate an error. This report 
 includes extended information about commodity holdings, including information 
 about the basis, gain, and return of individual commodities.</para>
+
+  <sect4 id="rpt-advport-capgains">
+    <title>Advanced Portfolio Capital Gains</title>
+    <para>
+      The Advanced Portfolio report doesn’t use the capital gain splits to
+      calculate capital gains. It calculates the gains from the various buy and
+      sell transactions in the account without regard to whether the gains and
+      losses are recorded or not. Any realized gain splits are ignored.
```

Comment:
Well you are quite correct. I did a test and splitting the Orphaned-Gains-AUD into 2 splits (Income:Capital Gain (Long Term):STK and Income:Capital Gains (Short Term): STK) made no difference to the Income or anything else on the APR.
I don't understand why Mike said:

> realized gains MUST be recorded as two splits

(capitalisation of MUST is mine). I thought the reason must be the previous sentence:

> They look a lot like income and the report goes to some lengths to make sure they aren’t counted that way.

I know that was an assumption but a logical one I thought. That was why I added comment 6 which he didn't contradict...   I don't want to give Mike a hard time - I really appreciate his work :-)